.github/publish-test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# This is a script meant to be used by Github Actions to test out pubsys related cargo-make tasks
+
+required_env() {
+  local env="${1:?}"
+  local value="${2}"
+  if [ -z "${value}" ]; then
+    echo "ERROR: Environment variable ${env} is required to be set" >&2
+    exit 2
+  fi
+}
+
+required_env "VARIANT" "${VARIANT}"
+required_env "ARCH" "${ARCH}"
+required_env "GITHUB_WORKSPACE" "${GITHUB_WORKSPACE}"
+
+echo -e "\n=^..^=   =^..^=   =^..^=   Create TUF repository from Bottlerocket build artifacts   =^..^=   =^..^=   =^..^=\n"
+cargo make -e BUILDSYS_VARIANT="${VARIANT}" -e BUILDSYS_ARCH="${ARCH}" repo
+mv build/repos/default build/repos/test
```

Comment:
There's currently a mix of unprefixed paths and GITHUB_WORKSPACE-prefixed paths which, to me, raises questions about the expected working directory.  I think if we `cd` to GITHUB_WORKSPACE at the start, we wouldn't need to reference it anywhere else, and it would clarify the purpose?

Reply:
```suggestion
# refresh-repo will fail if the output directory already exists, so we move it to a known path.
mv build/repos/default build/repos/test
```
(separately)